drivers/w1/w1_ds2477_85_common.h

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (c) 2022 Thomas Stranger
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_W1_DS2477_DS2485_COMMON_H_
+#define ZEPHYR_DRIVERS_W1_DS2477_DS2485_COMMON_H_
+
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/w1.h>
+#include <zephyr/kernel.h>
+
+/* memory specific commands */
+#define CMD_WR_MEM		 0x96
+#define CMD_RD_MEM		 0x44
+#define CMD_SET_PAGE_PROTECT	 0xc3
+#define CMD_RD_STATUS		 0xAA
+/* configuration specific commands */
+#define CMD_SET_I2C_ADDR	 0x75
+#define CMD_RD_W1_PORT_CFG	 0x52
+#define CMD_WR_W1_PORT_CFG	 0x99
+#define CMD_MASTER_RESET	 0x62
+/* 1-Wire specific commands */
+#define CMD_W1_SCRIPT		 0x88
+#define CMD_W1_BLOCK		 0xAB
+#define CMD_RD_BLOCK		 0x50
+#define CMD_WR_BLOCK		 0x68
+#define CMD_SEARCH		 0x11
+#define CMD_FULL_CMD_SEQ	 0x57
+/* crc16 specific commands */
+#define CMD_COMPUTE_CRC		 0xCC
+
+/* i2c command overhead len */
+#define CMD_OVERHEAD_LEN         2U
+/* memory specific commands' data length */
+#define CMD_WR_MEM_LEN		 33U
+#define CMD_RD_MEM_LEN		 1U
+#define CMD_SET_PAGE_PROTECT_LEN 2U
+#define CMD_RD_STATUS_LEN	 1U
+/* configuration specific commands */
+#define CMD_SET_I2C_ADDR_LEN	 1U
+#define CMD_RD_W1_PORT_CFG_LEN	 1U
+#define CMD_WR_W1_PORT_CFG_LEN	 3U
+/* 1-Wire specific commands */
+#define CMD_W1_SCRIPT_LEN	 1U
+#define CMD_W1_BLOCK_LEN	 1U
+#define CMD_RD_BLOCK_LEN	 1U
+#define CMD_WR_BLOCK_LEN	 1U
+#define CMD_SEARCH_LEN		 2U
+#define CMD_FULL_CMD_SEQ_LEN	 9U
+/* crc16 specific commands */
+#define CMD_COMPUTE_CRC_LEN	 1U
+
+/* I2C communication result bytes */
+#define DS2477_88_RES_SUCCESS	    0xAA
+#define DS2477_88_RES_INVALID_PARAM 0x77
+#define DS2477_88_RES_COMM_FAILURE  0x22
+#define DS2477_88_RES_RESET_FAILURE 0x22
+#define DS2477_88_RES_NO_PRESENCE   0x33
+#define DS2477_88_RES_WP_FAILURE    0x55
+
+/* primitive commands, executable via the script command */
+#define SCRIPT_OW_RESET	      0x00
+#define SCRIPT_OW_WRITE_BIT   0x01
+#define SCRIPT_OW_READ_BIT    0x02
+#define SCRIPT_OW_WRITE_BYTE  0x03
+#define SCRIPT_OW_READ_BYTE   0x04
+#define SCRIPT_OW_TRIPLET     0x05
+#define SCRIPT_OW_OV_SKIP     0x06
+#define SCRIPT_OW_SKIP	      0x07
+#define SCRIPT_OW_READ_BLOCK  0x08
+#define SCRIPT_OW_WRITE_BLOCK 0x09
+#define SCRIPT_OW_DELAY	      0x0A
+#define SCRIPT_OW_PRIME_SPU   0x0B
+#define SCRIPT_OW_SPU_OFF     0x0C
+#define SCRIPT_OW_SPEED	      0x0D
+
+/* port configuration register offsets */
+#define PORT_REG_MASTER_CONFIGURATION	0x00
+#define PORT_REG_STANDARD_SPEED_T_RSTL	0x01
+#define PORT_REG_STANDARD_SPEED_T_MSI	0x02
+#define PORT_REG_STANDARD_SPEED_T_MSP	0x03
+#define PORT_REG_STANDARD_SPEED_T_RSTH	0x04
+#define PORT_REG_STANDARD_SPEED_T_W0L	0x05
+#define PORT_REG_STANDARD_SPEED_T_W1L	0x06
+#define PORT_REG_STANDARD_SPEED_T_MSR	0x07
+#define PORT_REG_STANDARD_SPEED_T_REC	0x08
+#define PORT_REG_OVERDRIVE_SPEED_T_RSTL 0x09
+#define PORT_REG_OVERDRIVE_SPEED_T_MSI	0x0a
```

Comment:
The defines before these were using capitalized letters

Reply:
now all uniformly use lower-case.